Tools/Scripts/webkitdirs.pm

```diff
@@ -2946,6 +2950,47 @@ sub cmakeArgsFromFeatures(\@;$)
     return @args;
 }
 
+sub vcpkgArgsFromFeatures(\@;$)
+{
+    my ($featuresArrayRef, $enableExperimentalFeatures) = @_;
+
+    my @args;
+    my $avif = 0;
```

Comment:
Are these intended to be the default values? It seems unfortunate to encode a default in an additional place.

Reply:
Yep there really isnt any way around that without parsing the CMake files.